src/VisualStudio/Core/Def/EditorConfigSettings/CodeStyle/View/ColumnDefinitions/CodeStyleCategoryColumnDefinition.cs

```diff
@@ -50,7 +50,7 @@ public CodeStyleCategoryColumnDefinition()
         public override IEntryBucket? CreateBucketForEntry(ITableEntryHandle entry)
         {
             var categoryName = GetCategoryName(entry);
-            return categoryName is not null ? new StringEntryBucket(categoryName) : null;
+            return categoryName is not null ? new StringEntryBucket(categoryName, tooltip: categoryName) : null;
```

Comment:
Unrelated to editorconfig:

Callers of `ImageEntryBucket` don't provide tooltip and use the default parameter value `null`, is that an issue?

https://github.com/dotnet/roslyn/blob/26ac760823603fb0be65db030ef7d17bcda356db/src/VisualStudio/Core/Def/UnusedReferences/Dialog/UnusedReferencesTableProvider.ColumnDefinitions.cs#L347-L351